packages/commonwealth/server-test.ts

```diff
@@ -218,7 +218,6 @@ const rollbar = new Rollbar({
 setupErrorHandlers(app, rollbar);
 setupServer();
 
-export { resetDatabase } from './test/util/resetDatabase';
```

Comment:
This is the other change. We remove this so that when we call resetDatabase, we don't need to spin up a whole express server.